packages/wallets/src/features/cfd/screens/ChangePassword/MT5ChangePasswordScreens.tsx

```diff
@@ -15,6 +16,32 @@ const MT5ChangePasswordScreens: React.FC<MT5ChangePasswordScreensProps> = ({ pla
     const handleClick = (nextScreen: TChangePasswordScreenIndex) => setActiveScreen(nextScreen);
 
     const { hide } = useModal();
+    const { data } = useSettings();
+    const { mutate } = useVerifyEmail();
+    const { data: activeWallet } = useActiveWalletAccount();
+
+    let redirectTo = platform === 'mt5' ? 1 : 2;
+
+    if (activeWallet?.is_virtual) {
+        redirectTo = Number(`${redirectTo}1`);
+    } else {
+        redirectTo = Number(`${redirectTo}0`);
+    }
+
+    const handleSendEmail = async () => {
```

Comment:
Just make sure to add error handling as well

Reply:
Next PR 😃 🙏🏼